src/VolumeDrawable.ts

```diff
@@ -613,23 +614,31 @@ export default class VolumeDrawable {
   }
 
   onStartControls(): void {
-    this.PT && this.pathTracedVolume && this.pathTracedVolume.onStartControls();
+    if (this.volumeRendering instanceof PathTracedVolume) {
```

Comment:
this should be `if(rendermode is pathtrace) then this.volumeRendering.onStartControls()` OR simply `this.volumeRendering.onStartControls()` and same for onChangeControls and onEndControls